examples/i18n/src/lib.rs

```diff
@@ -0,0 +1,179 @@
+// Some Clippy linter rules are ignored for the sake of simplicity.
+#![allow(clippy::needless_pass_by_value, clippy::trivially_copy_pass_by_ref)]
+
+use seed::{prelude::*, *};
+
+use fluent::{FluentArgs, FluentBundle, FluentResource, FluentValue};
+use unic_langid::LanguageIdentifier;
+
+use enum_iterator::IntoEnumIterator;
+use std::borrow::Borrow;
+
+mod resource;
+
+// ------ ------
+//     Init
+// ------ ------
+
+fn init(_: Url, _: &mut impl Orders<Msg>) -> Model {
+    let mut model = Model::default();
+    change_lang(&mut model);
+    model
+}
+
+// ------ ------
+//     Model
+// ------ ------
+#[allow(non_camel_case_types)]
+#[derive(Clone, IntoEnumIterator, PartialEq)]
+enum Lang {
+    en_US,
+    de_DE,
+}
+impl Lang {
+    fn id(&self) -> &str {
+        match self {
+            Lang::en_US => "en-US",
+            Lang::de_DE => "de-DE",
+        }
+    }
+    fn label(&self) -> String {
+        match self {
+            Lang::en_US => "English (US)".to_string(),
+            Lang::de_DE => "Deutsch (Deutschland)".to_string(),
+        }
+    }
+}
+
+pub struct Model {
+    lang: Lang,
+    resource: FluentBundle<FluentResource>,
+}
+
+impl Default for Model {
```

Comment:
Remove `Default`. See review comments for `init` function above.

Reply:
Done.